stackdriver/client.go

```diff
@@ -82,6 +82,16 @@ type recoverableError struct {
 // version.* is populated for 'promu' builds, so this will look broken in unit tests.
 var userAgent = fmt.Sprintf("StackdriverPrometheus/%s", version.Version)
 
+var counter = 0
+var ips = []string{"199.36.153.4:443", "199.36.153.5:443",
+	"199.36.153.6:443", "199.36.153.7:443"}
+
+func ChooseIPs() string {
```

Comment:
This function isn't thread safe because of the global variable. Does it need to be thread safe (what's the Dialer contract?) If yes, consider an AtomicCounter: https://gobyexample.com/atomic-counters